rfc/src/rfcs/0013-list.md

```diff
@@ -179,4 +192,4 @@ fn check() {
 
 See [this blog post](https://model-checking.github.io/kani-verifier-blog/2022/10/27/using-kani-with-the-bolero-property-testing-framework.html) for more information.
 
-There's no easy way for us to know whether a harness comes from Bolero, since Bolero takes care of rewriting the test to use Kani syntax and invoking the Kani engine. By the time the harness gets to Kani, there's no way for us to tell it apart from a regular harness. Fixing this would require some changes to our Bolero integration.
\ No newline at end of file
+There's no easy way for us to know whether a harness comes from Bolero, since Bolero takes care of rewriting the test to use Kani syntax and invoking the Kani engine. By the time the harness gets to Kani, there's no way for us to tell it apart from a regular harness. Fixing this would require some changes to our Bolero integration.
```

Comment:
I'm not sure I follow the comment here. Bolero harnesses use the `kani::proof` attribute just like any other proof. Is the aim to distinguish bolero harnesses from kani harnesses?

Reply:
Ah I just expanded the paragraph above and that's exactly what the doc says:

> It would be nice to differentiate between regular Kani harnesses and Bolero harnesses. Bolero harnesses invoke Kani using conditional compilation

I'm open to any suggestions on how to improve this.

Reply:
In theory, I think we can look at the list of functions reachable from the harness and determine if it uses bolero. I'm not sure the extra overhead is worth it though.

Reply:
Yeah I'm not sure it's really worth distinguishing a bolero harness vs a vanilla Kani one... It's not super clear to me what one would actually do with that information.